src/example.c

```diff
@@ -1,26 +1,57 @@
 #include "libuvc/libuvc.h"
 #include <stdio.h>
+#include <unistd.h>
 
 /* This callback function runs once per frame. Use it to perform any
  * quick processing you need, or have it put the frame into your application's
  * input queue. If this function takes too long, you'll start losing frames. */
 void cb(uvc_frame_t *frame, void *ptr) {
   uvc_frame_t *bgr;
+  uvc_frame_t *rgb;
   uvc_error_t ret;
+  enum uvc_frame_format *frame_format = (enum uvc_frame_format *)ptr;
+  FILE *fp;
+  static int jpeg_count = 0;
+  static const char *H264_FILE = "iOSDevLog.h264";
+  static const char *MJPEG_FILE = ".jpeg";
```

Comment:
These two constants are not actually used, why not remove them altogether?

Reply:
comment unused code #143